config.js

```diff
@@ -1,4 +1,4 @@
 export default {
   REQUEST_THROTTLE_MS: 5000,
-  REQUEST_TIMEOUT_MS: 5000
+  REQUEST_TIMEOUT_MS: 5000,
```

Comment:
Welcome to the world of comma dangle.

It's sorta ugly but it's pretty useful when 90% of the code consists of objects.